multiversx_sdk_cli/contracts.py

```diff
@@ -78,19 +79,19 @@ def deploy(self, owner: Account, arguments: List[Any], gas_price: int, gas_limit
 
         tx = Transaction(
             chain_id=chain,
-            sender=owner.address,
-            receiver=Address.from_bech32(ADDRESS_ZERO_BECH32),
+            sender=owner.address.to_bech32(),
```

Comment:
Here can we use the new factory instead? Can be done now or in a next PR.

Reply:
Same for other features in `contracts.py`.

Reply:
Indeed. Was thinking about it, but I wanted to make sure everything works as expected. Will refactor to use the factory in a next PR.